README.md

```diff
@@ -1,53 +1,102 @@
-# Open Source Project Template
+﻿# Uno Platform Application Template
```

Comment:
This isn't an Uno application.

Reply:
Yes working on it, this was caused when copying to documentation from Uno Application Template.